barretenberg/cpp/src/barretenberg/plonk_honk_shared/execution_trace/ultra_execution_trace.hpp

```diff
@@ -37,6 +37,12 @@ template <typename T> struct UltraTraceBlockData {
                          poseidon2_internal, overflow };
     }
 
+    auto get() const
+    {
+        return RefArray{ pub_inputs,         lookup,  arithmetic, delta_range, elliptic, aux, poseidon2_external,
```

Comment:
weird formatting?

Reply:
looks funny yeah but its just aligning entries in the top line to the bottom line